src/BaselineUtils.cc

```diff
@@ -1331,15 +1429,16 @@ getSignificantEdgePixels(ImagePtrT img,
                          ImagePixelT thresh) {
     LOG_LOGGER _log = LOG_GET("meas.deblender.getSignificantEdgePixels");
 
-    sfoot->normalize();                 // Algorithms require spans sorted by y
 
-    PTR(det::Footprint) significant(new det::Footprint(sfoot->getPeaks().getSchema()));
+    auto significant = std::make_shared<det::Footprint>();
+    significant->setPeakSchema(sfoot->getPeaks().getSchema());
 
     int const x0 = img->getX0(), y0 = img->getY0();
-    CONST_PTR(det::Footprint) const edges = sfoot->findEdgePixels();
-    for (det::Footprint::SpanList::const_iterator ss = edges->getSpans().begin();
-         ss != edges->getSpans().end(); ++ss) {
-        geom::Span const& span = **ss;
+    std::shared_ptr<geom::SpanSet> edgeSpans = sfoot->getSpans()->findEdgePixels();
+    std::vector<geom::Span> tmpSpans;
+    for (geom::SpanSet::const_iterator ss = edgeSpans->begin();
+         ss != edgeSpans->end(); ++ss) {
```

Comment:
no, I don't know what happened there, it didn't even need to be on a different line, changed now